Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Dashboard/Management/DriverCoins.hs

```diff
@@ -82,6 +84,10 @@ bulkUpdateByDriverId merchantId merchantOpCityId driverId eventFunction coinsVal
       uuid <- generateGUIDText
       let expiryTime = fmap (\expirationTime -> UTCTime (utctDay $ addUTCTime (fromIntegral expirationTime) now) 0) mbexpirationTime
           status_ = if coinsValue > 0 then Remaining else Used
+      vehCategory <-
```

Comment:
In most the places in data vehicleCategory is nothing, so instead of that using vehicleVariant convert it into vehicleCategory